src/main/java/org/intellij/sdk/codesync/clients/CodeSyncWebSocketClient.java

```diff
@@ -171,7 +179,12 @@ public void sendDiffs(ArrayList<Pair<Integer, DiffFile>> diffsToSend, DataTransm
                 Long statusCode = (Long) response.get("status");
                 String diffFilePath = (String) response.get("diff_file_path");
                 if (statusCode != 200) {
-                    CodeSyncLogger.critical(String.format("Diff upload failed with error: %s.", response.get("error")));
+                    if (statusCode == Constants.ErrorCodes.DIFFS_LIMIT_REACHED) {
```

Comment:
Authenticate does not return 402, diff-send returns 402.  I think this logic should go under `.sendDiffs`

Reply:
@Basit-qc I think the functionality is already in the `.sendDiffs`, authentication happens in another step and this is the status code returned by the diffs processing endpoint.

Reply:
Got it, thanks